psi4/driver/driver_findif.py

```diff
@@ -1017,65 +1126,371 @@ def init_string(data):
+        self.metameta['irrep'] = data.pop('findif_irrep', -1)
+
+        if self.metameta['mode'] == '1_0':
+            self.metameta['proxy_driver'] = 'energy'
+            self.findifrec = gradient_from_energies_geometries(self.molecule,
+                                                               stencil_size=findif_stencil_size,
+                                                               step_size=findif_step_size,
+                                                               t_project=translations_projection_sound,
+                                                               r_project=r_project_grad)
+
+        elif self.metameta['mode'] == '2_1':
+            self.metameta['proxy_driver'] = 'gradient'
+            self.findifrec = hessian_from_gradients_geometries(self.molecule,
+                                                               freq_irrep_only=self.metameta['irrep'],
+                                                               stencil_size=findif_stencil_size,
+                                                               step_size=findif_step_size,
+                                                               t_project=translations_projection_sound,
+                                                               r_project=r_project_hess)
+
+        elif self.metameta['mode'] == '2_0':
+            self.metameta['proxy_driver'] = 'energy'
+            self.findifrec = hessian_from_energies_geometries(self.molecule,
+                                                              freq_irrep_only=self.metameta['irrep'],
+                                                              stencil_size=findif_stencil_size,
+                                                              step_size=findif_step_size,
+                                                              t_project=translations_projection_sound,
+                                                              r_project=r_project_hess)
+
+        ndisp = len(self.findifrec["displacements"]) + 1
+        info = f""" {ndisp} displacements needed ...\n"""
+        core.print_out(info)
+        logger.debug(info)
+
+        # var_dict = core.variables()
+        packet = {
+            "molecule": self.molecule,
+            "driver": self.metameta['proxy_driver'],
+            "method": self.method,
+            "basis": data["basis"],
+            "keywords": data["keywords"] or {},
+        }
+        if 'cbs_metadata' in data:
+            packet['cbs_metadata'] = data['cbs_metadata']
+        passalong = {k: v for k, v in data.items() if k not in packet}
+        passalong.pop('ptype', None)
+
+        self.task_list["reference"] = self.computer(**packet, **passalong)
+
+        parent_group = self.molecule.point_group()
+        for label, displacement in self.findifrec["displacements"].items():
+            clone = self.molecule.clone()
+            clone.reinterpret_coordentry(False)
+            #clone.fix_orientation(True)
```

Comment:
Why do we need one but not the other?

Reply:
It's usually fix_com, fix_orientation that go together, right? those two get set upon molecule validation in the class init. I probably kept the comment because line was active in the original src, and I wasn't sure if there were circs where it was needed again.